airflow-core/src/airflow/models/dag.py

```diff
@@ -516,13 +518,13 @@ def get_paused_dag_ids(dag_ids: list[str], session: Session = NEW_SESSION) -> se
         :param session: ORM Session
         :return: Paused Dag_ids
         """
-        paused_dag_ids = session.execute(
+        result = session.execute(
             select(DagModel.dag_id)
             .where(DagModel.is_paused == expression.true())
             .where(DagModel.dag_id.in_(dag_ids))
         )
 
-        paused_dag_ids = {paused_dag_id for (paused_dag_id,) in paused_dag_ids}
+        paused_dag_ids = {paused_dag_id for (paused_dag_id,) in result}
```

Comment:
I think this can just be

```python
paused_dag_ids = set(
    session.scalars(
        select(DagModel.dag_id)
        .where(DagModel.is_paused == expression.true())
        .where(DagModel.dag_id.in_(dag_ids))
    )
)
```